### DISCUSSION TITLE
提一些关于项目持续性小建议

### DISCUSSION THREAD

**Discussion by AH-dark** [lsky-org/lsky-pro#402] [想法]
1. Gravatar 源这个建议交给用户自行选择，毕竟作为开发者是很难保障一个源稳定高速的，这也可以降低日后维护成本。
2. Commit Log 建议写得规范些，可以参考一些文档，例如 <https://blog.csdn.net/qq_27445903/article/details/78092757>
3. 建议尽可能把页面做成前后端分离，减小些性能损耗
4.关于页面语言，局限于中文可能不太适合日后发展。

**Comment by 0xxb** [comment]
感谢你的建议，会酌情考虑。
